CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## 💡 Enhancements 💡
+
+- `datadogexporter` increase default timeout to 30 seconds (#6131)
```

Comment:
The change is updating the timeout to 15 seconds, not 30 correct?

Reply:
🤦 sorry about that, should be fixed now